### PR TITLE
chore(docs) Small formatting changes for Kafka-related content for better SEO

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: blog/2022-06-07-data-pipeline-with-kafka-and-questdb.md [hunk 1]
```diff
@@ -199,20 +199,19 @@ services:
     ports:
       - "9000:9000"
       - "8812:8812"
-
 ```
 
 In sequential order, this Docker Compose file installs Confluent-managed Kafka
 tools,
-[Zookeeper](https://www.cloudkarafka.com/blog/cloudkarafka-what-is-zookeeper.html),
+[Zookeeper](https://www.cloudkarafka.com/blog/cloudkarafka-what-is-zookeeper.html/),
```
Comment (pswu11) [inline blog/2022-06-07-data-pipeline-with-kafka-and-questdb.md:206]: no need to add / for external links i recon - this link is 404

File: blog/2022-06-07-data-pipeline-with-kafka-and-questdb.md [hunk 2]
```diff
@@ -310,7 +309,7 @@ print('Initialized Kafka producer at {}'.format(dt.datetime.utcnow()))
 # Creating a continuous loop to process the real-time data
 while True:
     # API request
-    uri = 'http://api.coincap.io/v2/assets/ethereum'
+    uri = 'http://api.coincap.io/v2/assets/ethereum/'
```
Comment (pswu11) [inline blog/2022-06-07-data-pipeline-with-kafka-and-questdb.md:312]: tho it probably won't matter but I'll suggest to not touch the code?

File: blog/2022-06-07-data-pipeline-with-kafka-and-questdb.md [hunk 3]
```diff
@@ -199,20 +199,19 @@ services:
...
 In sequential order, this Docker Compose file installs Confluent-managed Kafka
 tools,
-[Zookeeper](https://www.cloudkarafka.com/blog/cloudkarafka-what-is-zookeeper.html),
+[Zookeeper](https://www.cloudkarafka.com/blog/cloudkarafka-what-is-zookeeper.html/),
 and
 [Kafka broker](https://developer.confluent.io/learn-kafka/apache-kafka/brokers/),
 which manage the connections and processes in the Kafka ecosystem. Then, it
 installs a
-[JDBC Connector](https://docs.confluent.io/kafka-connect-jdbc/current/index.html)
+[JDBC Connector](https://docs.confluent.io/kafka-connect-jdbc/current/index.html/)
```
Comment (pswu11) [inline blog/2022-06-07-data-pipeline-with-kafka-and-questdb.md:211]: adding `/` makes it goes back to the overview page, not the kafka connect page we want.

**Comment by amyshwang** [comment]
As discussed - closing for now as we want more overall planning for SEO.

**Pull request closed without merging**